### ISSUE TITLE
Formatter automatically adds space around "-" inside HTML attributes

### ISSUE THREAD

**Issue by ralphjsmit** [Stillat/blade-parser-typescript#70]
A few days ago @cheesegrits installed the plugin on of my projects. Really liking it so far!

One thing I noticed (which actually broke one of my packages 🙃) is that it added a space around the "-" in an HTML attribute, see line 60:

<img width="1289" alt="Space" src="https://github.com/Stillat/blade-parser-typescript/assets/59207045/f07a9d22-469d-43a3-a9d6-f548f4ff3fb1">

Hereby the original code:

```blade
<div
    class="relative group"
    @if($isMultiple && $isReorderable)
        x-sortable-handle
     x-sortable-item="image-{{ $image->id }}"
     @endif
>
```

(I also would have liked if both `x-sortable-*` directives would be indented in the same way but perhaps related to the other recent issues from Hugh.)

Would be great if this could be fixed. Thanks!

**Comment by JohnathonKoster** [comment]
Resolved in `1.6.9` 🙂

**Comment by ralphjsmit** [comment]
Thanks!
